### PR TITLE
Feature icingaweb2 modules

### PR DESCRIPTION
**Pull request by bartjanssens92**
This commit adds a dynamic include for adding modules to the icingaweb2 interfaces.

As first example the director module is added.
It can be included by using a new parameter of the icingaweb2 class names modules.

### CODE REVIEW

File: manifests/alerting/icingaweb2/director.pp [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+  postgresql::server::db { $db_name:
+    user     => $db_username,
+    password => $db_password,
+  }
+
+  package { 'git':
+    ensure => present,
+  }
+
```
Comment (attachmentgenie) [inline manifests/alerting/icingaweb2/director.pp]: its probably best to place this in manifests/alerting/icingaweb2 in the modules loop as the requirement will hold true for all modules.

also by including it as follows their wont be conflicts with other classes including git e.g. manifests/testing/development 

ensure_resource('package', 'git', {'ensure' => 'present'})

File: manifests/alerting/icingaweb2/director.pp [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  String $api_user     = $::profiles::alerting::icingaweb2::api_user,
+  String $api_password = $::profiles::alerting::icingaweb2::api_password,
+  String $api_host     = 'localhost',
+) inherits profiles::alerting::icingaweb2 {
+
+  postgresql::server::db { $db_name:
+    user     => $db_username,
+    password => $db_password,
+  }
+
```
Comment (attachmentgenie) [inline manifests/alerting/icingaweb2/director.pp]: i prefer not to use resources outside of their unit of work e.g manifests/database/postgresql in this case. This does put the oneous on the developer/operator to include the correct hiera KV pairs

you ll find that icinga2.pp and icingaweb2.ppp dont include code to create the db's either

**Comment by bartjanssens92** [comment]
Moved the package resource and remove the db creation.

**Comment by attachmentgenie** [comment]
thanks!

**Pull request merged**